### ISSUE TITLE
green background: "can reach server" - not working correctly

### ISSUE THREAD

**Issue by tmcolby** [miyachiamericaeurope/amya-status-display#15]
we currently test if amya-publish-pickled.service is running, and if so, we assume we "can reach the server".  this is not an accurate way to determine that state however.  when a node is unclaimed by a server, this service would be dead (not running) and so would appear to work properly for this case.  once the node is claimed by a server, this service starts and stays running even if the service is unable to reach the hostname of the server (it just keeps retrying).

long story short - we need a different way to determine "can reach server" state
